lang/python/wiredtiger.i

```diff
@@ -55,6 +55,14 @@ from packing import pack, unpack
 ## @endcond
 %}
 
+/*
+ * Tell SWIG about types that won't see.  We don't need to be exact in our typing here,
```

Comment:
"Tell SWIG about types that won't see" doesn't parse for me.  But I know almost nothing about SWIG, so maybe this is just going over my head.

Reply:
Fixed, and added more to the comment.